pkg/controller/storagecluster/reconcile.go

```diff
@@ -62,6 +66,15 @@ osd_memory_target_cgroup_limit_ratio = 0.5
 	ReconcileStrategyManage ReconcileStrategy = "manage"
 	// ReconcileStrategyStandalone also means never reconcile (NooBaa)
 	ReconcileStrategyStandalone ReconcileStrategy = "standalone"
+
+	// DeviceTypeSSD represents the DeviceType SSD
+	DeviceTypeSSD DeviceType = "ssd"
+
+	// DeviceTypeHDD represents the DeviceType HDD
+	DeviceTypeHDD DeviceType = "hdd"
+
+	// DeviceTypeNVMe represents the DeviceType NVMe
+	DeviceTypeNVMe DeviceType = "nvme"
```

Comment:
Since the variables have `DeviceType` prefix, I don't see the benefit for using `DeviceType` alias for `string` type.

We can either do `SSD DeviceType = "ssd"` or `DeviceTypeSSD string = "ssd"`

Reply:
I did this because it has been done similar for adding const and naming just SSD or HDD fails for golint test

Reply:
Like in Line 44 53 57,  `DeviceTypeSSD = "ssd"` should be enough too.

Reply:
done